mlflow/models/evaluation/validation.py

```diff
@@ -0,0 +1,182 @@
+class MetricThreshold:
+    """
+    This class allows users to define metric thresholds for Model Validation. Allowed thresholds are
+    - threshold
+    - min_absolute_change
+    - min_relative_change
```

Comment:
Can we make sure this renders in the MLflow API docs and that there aren't any formatting errors? I suspect that we need newlines between the previous line and between the first parameter.

Reply:
![image](https://user-images.githubusercontent.com/25189433/183118035-0df32a72-dc2d-450d-bc42-53133879635e.png)